go.sum

```diff
@@ -1,3 +1,4 @@
+<<<<<<< Updated upstream
```

Comment:
Merge conflict?

Reply:
Dropped in https://github.com/keep-network/keep-ecdsa/pull/725/commits/1baa1eb72194531326ab2660aab344061a472893